lib/tasks/cleanup.rake

```diff
@@ -6,3 +6,10 @@ namespace :cleanup do
     TransientRegistrationCleanupService.run
   end
 end
+
+namespace :cleanup do
+  desc "Remove expired registrations older than 7 years from the database"
+  task old_regsitrations: :environment do
```

Comment:
Typo: "regsitrations". Maybe call it remove_expired_registrations for clarity?

Reply:
thank you - have corrected